yucca/training/trainers/YuccaLightningWrapper.py

```diff
@@ -0,0 +1,122 @@
+#%%
+import lightning as pl
+from batchgenerators.utilities.file_and_folder_operations import join
+from yucca.training.augmentation.YuccaAugmenter import YuccaAugmenter
+from yucca.training.data_loading.YuccaDataModule import YuccaDataModule
+from yucca.training.trainers.YuccaConfigurator import YuccaConfigurator
+from yucca.training.trainers.YuccaLightningModule import YuccaLightningModule
+from yucca.paths import yucca_preprocessed
+
+class YuccaLightningTrainer:
```

Comment:
I would prefer another name to avoid confusion with the lightning trainer. Maybe Yucca should not have a `trainer` concept anymore? Maybe something like `Operator` or `Manager`?

Reply:
Hmm, I like these actually